application/templates/forms/extended_hints/_chart_and_table_title.html

```diff
@@ -1,15 +1,15 @@
-<details class="govuk-details">
-  <summary class="govuk-details__summary">
-    <span class="govuk-details__summary-text">
-      What to include
-    </span>
-  </summary>
-  <div class="govuk-details__text">
-    <ul class="govuk-list govuk-list--bullet eff-list--hint">
-        <li>the type of value being shown (number, percentage, etc)</li>
-        <li>the base population</li>
-        <li>the particular characteristic that’s being measured</li>
-        <li>how the data is broken down</li>
-    </ul>
-  </div>
-</details>
+{% from "_macros/_details.html" import govukDetails %}
+
+{% set html %}
+  <ul class="govuk-list govuk-list--bullet eff-list--hint">
+    <li>the type of value being shown (number, percentage, etc)</li>
+    <li>the base population</li>
+    <li>the particular characteristic that’s being measured</li>
+    <li>how the data is broken down</li>
+  </ul>
+{% endset %}
+
+{{ govukDetails(
+  summaryText="Examples",
+  html=html
+) }}
```

Comment:
I'll wait for Kev to come along and chastise you about not having blank lines at the end of files... :)

Reply:
My work here is done.